docs/releasenotes/RELEASE-NOTES-5.0.0.md

```diff
@@ -0,0 +1,65 @@
+# Semantic MediaWiki 5.0.0
+
+Released on Octobar 24th, 2024.
+
+## Summary
+
+This is a [minor release](../RELEASE-POLICY.md). Thus, it contains no breaking changes, only bug fixes and new features.
```

Comment:
5.0.0 is still considered a minor release?

Reply:
Updated.